notebooks/community/vertex_endpoints/optimized_tensorflow_runtime/bert_optimized_online_prediction.ipynb

```diff
@@ -134,6 +134,7 @@
         "* Python 3\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    !gsutil rm -r $BUCKET_URI
</pre><p>We will also need to change this line to BUCKET_URI/vertex_bert_sample/</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/481/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='133'/>

Reply:
BUCKET_URI was created in this notebook, and has to be deleted in the end